.github/workflows/UpdatePropertiesTableInMSDocs.yaml

```diff
@@ -0,0 +1,101 @@
+
+      - name: Commit files changes to topic branch
+        id: commit_files_changes_to_topic_branch
+        run: |
+          cd azure-dev-docs-pr
+          if [[ ${{ steps.check-whether-topic-branch-exist.outputs.topic_branch_exists }} -eq 1 ]] ; then
+            echo "The branch ${{ steps.get_project_version_and_branch_name.outputs.branch_name }} already exists."
+            git checkout ${{ steps.get_project_version_and_branch_name.outputs.branch_name }}
+          elif [[ ${{ steps.check-whether-topic-branch-exist.outputs.topic_branch_exists }} -eq 0 ]] ; then
+            echo "The branch ${{ steps.get_project_version_and_branch_name.outputs.branch_name }} does not exist, create."
+            git checkout -b ${{ steps.get_project_version_and_branch_name.outputs.branch_name }}
+          fi
+          cp -f ../spring-cloud-azure-markdown-appdendix-generator/src/main/asciidoc/configuration-*.md articles/java/spring-framework/includes/spring-cloud-azure
+          if [[ -n "$(git status -s)" ]] ;then
+            echo '::set-output name=whether_to_create_pr::true'
+            git add ./articles/java/spring-framework/includes/spring-cloud-azure/configuration-*.md
+            git commit -m "Update the configuration properties for Spring Cloud Azure: ${{ steps.get_project_version_and_branch_name.outputs.project_version }}. This commit is created by GitHub Action: https://github.com/microsoft/spring-cloud-azure/actions"
+            git push "https://${{ secrets.UPDATE_PROPERTIES_TABLE_IN_MS_DOCS_GITHUB_USERNAME }}:${{ secrets.UPDATE_PROPERTIES_TABLE_IN_MS_DOCS_GITHUB_ACCESS_TOKEN }}@github.com/${{ secrets.UPDATE_PROPERTIES_TABLE_IN_MS_DOCS_GITHUB_USERNAME }}/azure-dev-docs-pr.git"
+          else
+            echo "No file changes, no commits."
+          fi
+
+      - name: Create Pull Request
+        if: ${{ steps.check-whether-topic-branch-exist.outputs.topic_branch_exists == '0' && steps.commit_files_changes_to_topic_branch.outputs.whether_to_create_pr == 'true' }}
+        uses: vsoch/pull-request-action@master
+        env:
+          PULL_REQUEST_TOKEN: ${{ secrets.UPDATE_PROPERTIES_TABLE_IN_MS_DOCS_GITHUB_ACCESS_TOKEN }}
+          PULL_REQUEST_REPOSITORY: MicrosoftDocs/azure-dev-docs-pr
+          PULL_REQUEST_TITLE: "Update the configuration properties for Spring Cloud Azure: ${{ steps.get_project_version_and_branch_name.outputs.project_version }}"
+          PULL_REQUEST_FROM_BRANCH: ${{ secrets.UPDATE_PROPERTIES_TABLE_IN_MS_DOCS_GITHUB_USERNAME }}:${{ steps.get_project_version_and_branch_name.outputs.branch_name }}
+          PULL_REQUEST_BRANCH: "main"
+          PULL_REQUEST_BODY: "Update the configuration properties for Spring Cloud Azure: ${{ steps.get_project_version_and_branch_name.outputs.project_version }}."
```

Comment:
Add more description. For example:
```
This PR is created by GitHub Actions. ${link_to_action_url} 
```
`${link_to_action_url}` can be got by this way: 
https://github.community/t/get-runs-url/16921/2